### ISSUE TITLE
keras.activations.sigmoid() fails when NN run @ C++

### ISSUE THREAD

**Issue by rep-stosw** [Dobiasd/frugally-deep#440]
Hi Tobias!

I made last activation layer Sigmoid, written as **keras.activations.sigmoid** . Model converts, but in C++ run fails:

> Loading json ... done. elapsed time: 0.000150 s
> Building model ... terminate called after throwing an instance of 'std::runtime_error'
>   what():  unknown layer type: Sigmoid
> make: *** [run] Error 9

Code fragment (minimal):

input_e = Input(shape=(Dim,1))
output_e = Conv1D(o, (12), strides=12, padding='same', use_bias=False, activation=None)(input_e)
d = Conv1DTranspose(1, (12), strides=12, padding='same', use_bias=False, activation=None)(output_e)
output_d = keras.activations.sigmoid(d)

c_encoder=Model(input_e,output_d,name="decoder")

I understand, that sigmoid can be add in last convolution layer, but another cases requires another sequence of layers.

Please, add such layer form.

**Comment by Dobiasd** [comment]
Ah, good remark. Thanks! 👍

Yeah, support for the `keras.activations....` layers is missing in frugally-deep so far.

I'll implement it. It should not take too long. 🧑‍🔧

Until then (in case you're in a hurry) you can replace

`keras.activations.sigmoid`

with

`keras.layers.Activation("sigmoid")`

which is already supported by frugally-deep.

**Comment by Dobiasd** [comment]
Hi, I just released [a new version of frugally-deep](https://github.com/Dobiasd/frugally-deep/releases/tag/v0.18.0).

It now supports `keras.activations.sigmoid` and a few other new ones. : )

**Comment by rep-stosw** [comment]
> Hi, I just released [a new version of frugally-deep](https://github.com/Dobiasd/frugally-deep/releases/tag/v0.18.0).

Thanks!
